src/App.js

```diff
@@ -1,18 +1,63 @@
 import React, { Component } from 'react';
 import logo from './logo.svg';
 import './App.css';
+import uuid from 'uuid';
+import Swipeable from './Swipeable.js';
+import Interactions from './Interactions.js';
+
 
 class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: 'unknown@mit.edu',
+      workshopCode: 'foo',
+      sessionId: uuid.v4()
+    };
+    this.onSwipeLeft = this.onSwipeLeft.bind(this);
+    this.onSwipeRight = this.onSwipeRight.bind(this);
+  }
+
+  // Describe context of the game session
+  session() {
+    const {email, workshopCode, sessionId} = this.state;
+    return {
+      email,
+      workshopCode,
+      sessionId,
+      clientTimestampMs: new Date().getTime(),
+      location: window.location.toString()
+    };
+  }
+
+  // Log an interaction to the server, along with context about the session
+  doLogInteraction(interaction) {
+    const session = this.session();
+    console.log('doLogInteraction', {interaction, session});
```

Comment:
This is just placeholder for now - this will write to the server in subsequent iterations.